packages/core/components/Puck/index.tsx

```diff
@@ -16,7 +16,7 @@ import { Plugin } from "../../types/Plugin";
 import { usePlaceholderStyle } from "../../lib/use-placeholder-style";
 
 import { SidebarSection } from "../SidebarSection";
-import { ChevronDown, ChevronUp, Globe, Sidebar } from "react-feather";
+import { ChevronDown, ChevronUp, Globe, PanelLeft } from "lucide-react";
```

Comment:
You've updated the import but missed the `Sidebar` uses further below (highlighted by the GitHub Actions comments)